spikeextractors/extraction_tools.py

```diff
@@ -562,4 +562,82 @@ def load_extractor_from_dict(d):
     extractor: RecordingExtractor or SortingExtractor
         The loaded extractor object
     '''
-    return BaseExtractor.load_extractor_from_dict(d)
\ No newline at end of file
+    return BaseExtractor.load_extractor_from_dict(d)
+
+
+def check_get_traces_args(func):
+    @wraps(func)
+    def corrected_args(*args, **kwargs):
+        # parse args and kwargs
+        if len(args) == 1:
+            recording = args[0]
```

Comment:
why do you play with arg wheras get_traces is kargs ?

Reply:
it can be args as well. there is no `*`

Reply:
It could be args as well as there is no `*`